components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/webextension/GeckoWebExtension.kt

```diff
@@ -29,7 +29,7 @@ import org.mozilla.geckoview.WebExtension.Action as GeckoNativeWebExtensionActio
  * Gecko-based implementation of [WebExtension], wrapping the native web
  * extension object provided by GeckoView.
  */
-@Suppress("TooManyFunctions")
+@Suppress("TooManyFunctions", "Deprecation")
```

Comment:
Let's put the `@Suppress("Deprecation")` on the actual inner constructor call, instead of the whole class, so we know exactly why were doing it. Let's also link to: https://github.com/mozilla-mobile/android-components/issues/6356